src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/FileUploadReviewViewModelToBulkUploadAddDraftApprenticeshipsRequestMapper.cs

```diff
@@ -26,10 +27,13 @@ public async Task<BulkUploadAddDraftApprenticeshipsRequest> Map(FileUploadReview
         {
             var cacheModel = await _cacheService.GetFromCache<FileUploadCacheModel>(source.CacheRequestId.ToString());
             var csVRecords = cacheModel.CsvRecords;
+            var fileUploadLogId = cacheModel.FileUploadLogId;
+
             await _cacheService.ClearCache(source.CacheRequestId.ToString(),nameof(FileUploadReviewViewModelToBulkUploadAddDraftApprenticeshipsRequestMapper));
             return new BulkUploadAddDraftApprenticeshipsRequest
             {
                 ProviderId = source.ProviderId,
+                FileUploadLogId = fileUploadLogId,
```

Comment:
WhenMappingFileUploadStartViewModelToBulkUploadRequestMapperTests.cs

is the testing class for this mapper. We should add a test in to ensure the log id is mapped as expected

Reply:
Added a test for this based on asset from cache id to apirequest param